app/controllers/search_controller.rb

```diff
@@ -1,59 +1,36 @@
 class SearchController < ApplicationController
-	before_filter :if_not_signed_in
+  before_filter :if_not_signed_in
```

Comment:
We should definitely reuse this code instead of redeclaring it in each controller. Thanks for bringing this up!